### ISSUE TITLE
Old Firefox logo needs to be updated in Add-ons web page

### ISSUE THREAD

**Issue by EdgardoRios** [mozilla/addons#712]
### Describe the problem and steps to reproduce it:
Old Firefox logo needs to be updated in Add-ons web page.
1. Go to an https://addons.mozilla.org/en-US/firefox/addon/ `myadd-on`
Please change `myadd-on` by `spanish-mexico-dictionary/`, or, I think you should be able to pick any other add-on for testing purposes.
2. Log in to Firefox Add-ons.
3. Scroll down until you find `More Information` div (on the left)
3. Find last option `Usage Statistics`
4. Click on [Visit stats dashboard](https://addons.mozilla.org/en-US/firefox/addon/spanish-mexico-dictionary/statistics/)

### What happened?
You will see the old Firefox logo at the top left of the screen.

### What did you expect to happen?
See the new Firefox logo, maybe even the the new brand for the site.

### Anything else we should know?
Attached you will see a screen capture of the header.
![addons](https://user-images.githubusercontent.com/18619087/39827773-8d8bbc8c-537e-11e8-9ea9-ec0404250649.png)

**Comment by jvillalobos** [comment]
The developer-facing sections of the site haven't been ported to the new design. This includes the statistics dashboard, which does overlap in some instances with user-facing pages. We haven't planned on porting those pages yet, so I don't think it's worth tracking this for now.

I filed https://github.com/mozilla/addons-server/issues/8224 to fix the icon, but the remaining work will be filed and tracked once we have a plan of porting of all those pages to the new style. Closing this one.
